.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+name: Build release files
+on:
+  workflow_dispatch:
+  push:
+    branches: [ develop ]
+    paths:
+      - 'src/ontology/imports/**_import.owl'
+      - 'src/ontology/components/**.owl'
```

Comment:
What about the edit file?

Reply:
Is your intention to have a release after each PR is merged? How about when two PRs are merged on the same day?

Reply:
you are right. maybe lets just remove the branch trigger altogether and instead only support `workflow_dispatch`